src/interp/interp.h

```diff
@@ -1246,6 +1241,8 @@ class Thread : public Object {
 
   RunResult StepInternal(Trap::Ptr* out_trap);
 
+  Thread* prev_thread_;
+
```

Comment:
I think this can be removed now?

Reply:
The `friend class Thread;` is still needed because the Thread constructor/destructor add/removes the Thread from the set in the storage. It could be removed if the `set` would be accessible for Thread somehow.